src/components/navigation.js

```diff
@@ -16,10 +17,14 @@ const Navigation = () => {
           <Link to='/blog/' activeClassName="active">Blog</Link>
         </NavItem>
       </NavLists>
-    </nav>
+    </NavContianer>
   );
 };
 
+const NavContianer = styled.nav`
+  width: calc(100% - 48px);
```

Comment:
Values like this 48px are called magic numbers because nobody apart from the original developer knows why the value is 48 and not 49 or 50 etc. If this is based on a padding value or something it should use the padding variable or something so that if someone changes the padding variable then it is also updated here.